lib/gui/app/components/drive-selector/controllers/drive-selector.js

```diff
@@ -75,17 +75,24 @@ module.exports = function (
       return $q.resolve(false)
     }
 
-    if (constraints.isDriveSizeRecommended(drive, selectionState.getImage())) {
-      return $q.resolve(true)
+    if (!constraints.isDriveSizeRecommended(drive, selectionState.getImage())) {
+      return WarningModalService.display({
+        confirmationLabel: 'Yes, continue',
+        description: [
+          messages.warning.unrecommendedDriveSize(selectionState.getImage(), drive),
+          'Are you sure you want to continue?'
```

Comment:
Perhaps this `'Are you sure you want to continue?'` should be folded into `messages.warning.unrecommendedDriveSize` ?